apps/web/src/SettingsRoutes.tsx

```diff
@@ -15,8 +16,13 @@ import { EmailSettings } from './pages/settings/tabs/EmailSettings';
 export const useSettingsRoutes = () => {
   const isInformationArchitectureEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_INFORMATION_ARCHITECTURE_ENABLED);
 
-  return isInformationArchitectureEnabled ? (
+  /*
+   * FIXME: reload doesn't work properly with conditional routes,
+   * so replace with isInformationArchitectureEnabled before merging
+   */
+  return true ? (
```

Comment:
I would like to merge this into the main `information-architecture` branch as it makes dev much easier for the time being